src/View/Cell.php

```diff
@@ -176,6 +176,17 @@ public function render($template = null)
         }
 
         $render = function () use ($template) {
+            try {
+                $reflect = new ReflectionMethod($this, $this->action);
+                $reflect->invokeArgs($this, $this->args);
```

Comment:
I'm glad to know I did not miss anything !
I'll try to add a test to cover this to avoid regressions in the future tonight.